### DISCUSSION TITLE
No information receive and MEM Error

### DISCUSSION THREAD

**Discussion by lam49** [Koenkk/zigbee2mqtt#9823] [General]
Hello everyone,

I work on a zigbee2mqtt project. All are installed and work fine during 4 days.

At 2021-11-23 16h30, i have no answer from my device.
So the next days, i try to communicate but nothing and after try to set premit joi "true", this message is coming :
`error 2021-11-24 09:01:21: Failed to call 'BridgeLegacy' 'onMQTTMessage' (Error: SREQ '--> ZDO - mgmtPermitJoinReq - {"addrmode":15,"dstaddr":65532,"duration":254,"tcsignificance":0}' failed with status '(0x10: MEM_ERROR)' (expected '(0x00: SUCCESS)')
    at Object.func (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:322:27)
    at Queue.executeNext (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32))`

I would like to know if you have information about this problem : No answer and Mem Error during set permit join.

I have try different ressources and my first system work fine so i don't undrstand why this one don't work. I'm sure my EndDevice have no problem.

Zigbee2mqtt version : 1.21.2
Zigbee-Herdsman version : 0.13.149
Coordinator CC2538 revision : 20201010
Coordinator : [Link](https://www.ti.com/tool/CC2538-CC2592EM-RD)
Type Stack : zStack30x

I'm intresting by any clue to understand the situation or solve the problem.

Best regards,

Lam

**Comment by Koenkk** [comment]
This means your coordinator has crashed, please create a ticket in the repo where you downloaded the firmware from. Reflashing should get  your coordinator up and running again.

**Reply by lam49** [reply]
Hey thanks for you replay Koenkk,

Could you tell me if request zigbee2mqtt/bridge/config/devices/get # which often occur can be a source of coordinator crash ?

**Reply by Koenkk** [reply]
Not specifically

**Reply by lam49** [reply]
ok because i have test this system during 2 weeks and everything was working well. So i don't understand what source can result of this type of default.

Changes that i am done this change brocker and i see that this brocker generate message on zigbee2mqtt/bridge/config/devices and zigbee2mqtt/bridge/log.

**Comment by lam49** [comment]
After turn OFF/ON my raspberry, the coordinator work well and message zigbee are received. 

The coordinator answer to zigbee2mqtt because no error messages coming but no zigbee message succeed to arrive too.
Why just when i set permit join error message coming ? In the zigbee2mqtt code wich condition fail resulat of a 0x10 MEM_ERROR ?

**Reply by Koenkk** [reply]
It happens on any command when corruption occurs in the coordinator (which is an issue in the firmware)

**Comment by lam49** [comment]
@Koenkk 
Thanks for this informations.
Can you advice me a coordinator for my project to replace mine please ? 
My project need to receive information from 60 devices and with no router.

**Reply by Koenkk** [reply]
See https://www.zigbee2mqtt.io/guide/adapters/#recommended

**Reply by lam49** [reply]
ok thanks
